src/common/Executor.cpp

```diff
@@ -919,22 +971,28 @@ void Executor::runKernel(KernelBase* kernel, bool print_kernel_name)
 
     if ( run_params.showProgress() ) {
       if ( kernel->hasVariantDefined(vid) ) {
-        getCout() << "   Running ";
+        getCout() << "\tRunning ";
       } else {
-        getCout() << "   No ";
+        getCout() << "\tNo ";
       }
       getCout() << getVariantName(vid) << " variant" << endl;
     }
 
     for (size_t tune_idx = 0; tune_idx < kernel->getNumVariantTunings(vid); ++tune_idx) {
-
-      if ( run_params.showProgress() ) {
-        getCout() << "     Running "
-                  << kernel->getVariantTuningName(vid, tune_idx) << " tuning";
+      std::string const& tuning_name = kernel->getVariantTuningName(vid, tune_idx);
+      if (find(tuning_names[vid].begin(), tuning_names[vid].end(), tuning_name) != tuning_names[vid].end()) { // Check if valid tuning
+        if ( run_params.showProgress() ) {
+          getCout() << "\t\tRunning "
+                    << tuning_name << " tuning";
+        }
+        kernel->execute(vid, tune_idx); // Execute kernel
+        if ( run_params.showProgress() ) {
+          getCout() << " -- " << kernel->getLastTime() << " sec." << endl;
+        }
       }
-      kernel->execute(vid, tune_idx);
-      if ( run_params.showProgress() ) {
-        getCout() << " -- " << kernel->getLastTime() << " sec." << endl;
+      else {
```

Comment:
put this all on one line

Reply:
[dda1cfa](https://github.com/LLNL/RAJAPerf/pull/333/commits/dda1cfa587df85ed0f1fab14b9c56bd920375979)